packit_service/worker/helpers/build/copr_build.py

```diff
@@ -537,10 +596,62 @@ def submit_copr_build(self, script: Optional[str] = None) -> Tuple[int, str]:
 
         return build.id, self.api.copr_helper.copr_web_build_url(build)
 
-    def handle_build_submit_error(self, ex) -> TaskResults:
+    def run_copr_build(self) -> TaskResults:
+        """
+        Run copr build using SRPM built by us.
+        """
+        self._report(
```

Comment:
Can't we get rid of it now?

I'd think we can get rid of the whole function :eyes:

Reply:
Resolved by d09fdac9d6c4ea65185670421c937e34f1e570a0